assets/src/edit-story/components/library/panes/media/media3p/categoryPill.js

```diff
@@ -74,6 +79,8 @@ const CategoryPill = ({
       role="tab"
       aria-selected={isSelected}
       data-testid="mediaCategory"
+      className="categoryPill"
```

Comment:
Optional: There's no one way to list them, but I notice that data and className attributes are typically listed closer to the top of the attribute list, for example:

 return (
    <PillContainer
      ref={ref}
      className="categoryPill"
      data-category-id={categoryId}
      data-testid="mediaCategory"
      :

Reply:
Done